api/reference/1.0/notes/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Overview
+---
+
+# Notes
+
+`Notes` are textual annotations you can add to Launch resources of these types:
+- `DataElements`
+- `RuleComponents`
+- `Rules`
+- `Extensions`
+- `Libraries`
+- `Properties`
+
+Those six types are collectively known as the `Notable` resources.
+
+When a `Notable` resource is deleted, its associated `Notes` are also deleted.
+
+## `Notes` are Comments
+Launch doesn't attach any meaning to `Notes`.
+Adding a `Note` won't change the behavior of your `Property`.
+
+You might use `Notes` to:
+- provide background information
+- function as to-do lists
+- pass along resource usage advice
+- give instructions to other team members
+- record historical context
+- remind yourself why this resource is built the way it is
+
+But Launch ignores `Notes` so you are free to use them any way you want.
+
+## Revisable Notables
+
+In the case of revisable types
+(`DataElements`, `RuleComponents`, `Rules`, and `Libraries`),
+any `Notes` must be created on the current (`head`)
+revision. They may not be attached to other revisions.
+
+However, `Notes` may be _read_ from revisions. In such cases, the endpoints
+will return only the `Notes` that existed before the creation of the revision.
+They provide a snapshot of the annotations as they were when the revision was cut.
+In contrast, reading `Notes` from the current (`head`) revision will return all
+its `Notes`.
+
+## Limitations
+
+Each `Note` is limited to 512 Unicode characters.
+
+`Notes` are not editable.
+The only way to delete them is to delete their corresponding `Notable`.
+
+`Notes` cannot be attached to non-`Notable` resources. Specifically,
+you cannot add `Notes` to
+`Adapters`, `AuditEvents`, `Builds`, `Callbacks`, `Companies`,
+`Environments`, `ExtensionPackages`, `Hosts`, or `Profiles`.
```

Comment:
This section is redundant. The notable resources were already outlined at the start of the document.

Reply:
Yes. But this section is the *Limitations* section, and I wanted to emphasize here that you can't just drop Notes on _every_ kind of thing, and make it crystal clear to the user that his favorite note target was not just overlooked. Kind of a pre-emptive answer to "Why can't I get the API to let me add Notes to my Adapter?"

With that understanding, do you still want me to pull it?

Reply:
I would remove it. Check with @thebenrob.

Reply:
This page is kinda functioning as the Guides section for now and will be the only place in docs (until the UI is done) where we describe the functionality here.  I think it's fine to be explicit about what resources they cannot be attached to.  I would say that we should make sure the list of non-Notable resources does not include any resources that are not already in the documentation (not sure on Profiles for example).

Reply:
[Profiles](https://developer.adobelaunch.com/api/reference/1.0/profiles/) are already in the documentation. And I've double-checked the rest, and believe they are all documented types.

I actually built that list by reading the [Reference/1.0](https://developer.adobelaunch.com/api/reference/1.0/profiles/#) menu choices out of my local Jekyll instance -- which, surprisingly, caused me to accidentally include 'Adapters' originally. There was an empty directory named 'adapters' on my system -- a holdover from before the rename to 'hosts' -- and that  caused an empty entry in the docs menu, which I blindly copied without thinking. So I carefully reviewed the list in my proposed documentation to make sure they were all legit.